src/ubuntu/22.04/crossdeps/Dockerfile

```
@@ -40,12 +40,5 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # install llvm-toolchain using official script
-RUN wget -O- https://apt.llvm.org/llvm.sh | bash -s -- 18 \
-        clang \
-        clang-tools \
-        liblldb-dev \
-        lld \
-        lldb \
-        llvm \
-        python3-lldb \
+RUN wget -O- https://apt.llvm.org/llvm.sh | bash -s -- 18 all \
```

@akoeplinger, alternative is a bit light-weight:

```suggestion
RUN wget -O- https://apt.llvm.org/llvm.sh | bash -s -- 18 \
    && apt-get install -y \
        clang-tools-18 \
        liblldb-18-dev \
        llvm-18 \
```
WDYT?

---

sounds good to me. does this include lld?

---

Yup, the default set has lld: https://github.com/opencollab/llvm-jenkins.debian.net/blob/879d62528f2ac4f9d122c260848a402ee8b4a2e7/llvm.sh#L174